src/content/docs/en/guides/integrations-guide/alpinejs.mdx

```diff
@@ -96,31 +96,52 @@ Once the integration is installed, you can use [Alpine.js](https://alpinejs.dev/
 
 Check our [Astro Integration Documentation][astro-integration] for more on integrations.
 
-## Limitations
+Note that you can use the CDN builds of plugins without any issues, just add them to the `head`:
 
-The Alpine.js integration does not give you control over how the script is loaded or initialized. If you require this control, consider [installing and using Alpine.js manually](https://alpinejs.dev/essentials/installation). Astro supports all officially documented Alpine.js manual setup instructions, using `<script>` tags inside of an Astro component.
+```diff lang="html"
+<head>
```

Comment:
Sorry for being so insistent on this, but I have to ask: Were you really able to successfully test this? :) From what I've read, this was one of the issues that users were struggling with in the Discord support thread: the Alpine plugins loaded via deferred script tags were not guaranteed to have finished loading before the Alpine integration would initialize Alpine, and were therefore not reliably picked up by Alpine.

Did you change anything in the code that makes this possible now?

If it's possible now, awesome! If so, I'd love to see where this `<head>` tag should be placed by adding more context, e.g. a file name tab and the surrounding code - users should be able to know where they should add this so that the plugin is available on all pages of their site.

Reply:
Sure no problem! Have a look at this e2e test https://github.com/withastro/astro/pull/9812, I think that's exactly the scenario you're mentioning. It seems to work without my recent changes

Reply:
It's awesome that you have actually added an E2E test! However, is this test actually effective in checking if the collapse plugin is working? From what I see, it's checking whether `<p id="foo" x-show="expanded" x-collapse>` has a class of `hidden`, and if not, it assumes the test has passed. But is this not the case even when the plugin is not loaded at all?

Reply:
I manually checked and yes, the collapse plugin adds a `hidden` attribute so that's a safe check

Reply:
Ok, thank you for confirming. I also checked it manually now, even while simulating a slow connection using devtools, and the plugin always worked. I wonder why it didn't for the users reporting issues on Discord.

So, the only thing that's left is providing a code block file name and a little bit of context in your example so that users reading it will know immediately where to put this.

Reply:
Tell me if that looks good enough to you!

Reply:
A lot better, thank you! I just left a comment regarding file naming (layout vs. page). It was actually already helpful that we talked about this because I didn't even have a layout in mind, but that's actually a great way to add the script to each page head!